packages/app/app/assets/css/globals.scss

```
@@ -36,6 +36,32 @@
     border-style: none;
   }
 }
+// Only show scrollbar when content actually exceeds screen height
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)